### ISSUE TITLE
Deep Biaffine Attention For Neural Dependency Parsing

### ISSUE THREAD

**Issue by Tahkare** [Tahkare/PhD-archive#3]
## TL;DR
This article introduces deep biaffine attention instead of bilinear layers in models for dependency parsing. They present it as a traditional affine transformation but using a (d * d) linear transformation instead of a weight matrix.

### Article Link
https://arxiv.org/pdf/1611.01734.pdf

### Author
T. Dozat & C. Manning

## Key Takeaways
* New layer to compute scores in dependency parsing

## Comments/ Questions
This can easily be applied to other types of parsing.
For r<sub>i</sub> and r<sub>j</sub> the outputs of a bi-LSTM at different positions, we have :
* h<sub>i</sub> = MLP(r<sub>i</sub>) and h<sub>j</sub> = MLP(r<sub>j</sub>)
* s(i,j) = h<sub>j</sub><sup>T</sup> W h<sub>i</sub> where W is a 3D matrix
